### PR TITLE
Update JWT sample code

### PR DESCRIPTION
**Pull request by alshamma**
1. Update code to use new documentOperations API for adding a layer
2. Refactor code to use JavaScript async/await

### CODE REVIEW

File: sample_code/jwt-sample-app/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 Adobe. All rights reserved.
+Copyright 2019 Adobe. All rights reserved.
```
Comment (macdonst) [inline sample_code/jwt-sample-app/index.js]: Should be 2018-2019

File: sample_code/jwt-sample-app/index.js [hunk 2]
```diff
@@ -33,32 +33,25 @@ const S3 = new AWS.S3({ apiVersion: '2006-03-01', signatureVersion: 'v4' })
 const statusRetryMillis = 200
 const statusRetries = 20
 
-writeStatus('1. STARTING')
-
 /* Given the settings in the config file, create a JWT */
-function generateJWT (cb) {
-  writeStatus('2. Generate JWT')
+async function generateJWT () {
```
Comment (macdonst) [inline sample_code/jwt-sample-app/index.js]: Consider replacing the generateJWT and requestServiceToken methods with the [@adobe/jwt-auth](https://www.npmjs.com/package/@adobe/jwt-auth) package.

```
const auth = require('@adobe/jwt-auth');
 
let token = await auth(config);
```

If you'd like I can send a PR on this.

File: sample_code/jwt-sample-app/index.js (hunk 1 above)
Comment (alshamma) [reply sample_code/jwt-sample-app/index.js]: 👍

File: sample_code/jwt-sample-app/index.js (hunk 2 above)
Comment (alshamma) [reply sample_code/jwt-sample-app/index.js]: Cool, I'll make this change.

Comment (macdonst) [reply sample_code/jwt-sample-app/index.js]: Let me know if you run into any issues. I wrote the package to make life for internal and external devs easier.

Comment (alshamma) [reply sample_code/jwt-sample-app/index.js]: Hmm, yes. I'll contact you offline.

Comment (alshamma) [reply sample_code/jwt-sample-app/index.js]: Works now.

**Comment by noahhoffman** [comment]
👍

**Pull request merged**